contracts/core/State.sol

```diff
@@ -204,6 +204,8 @@ contract State is Objects, ReentrancyGuard, Ownable {
 	mapping(address => mapping(address => uint256)) public specialRebates; // Special rate rebates for spesific pair -- if not set, then use the default one
 	bool public pause; //Flag to pause all protocol modules
 
+	uint256 internal tradingRebateRewardsBasisPoint;
```

Comment:
all variables which are added should be documented.
e.g.: 
Defines the portion of the trading fees which is to be paid out in a liquid form in basis points (0.01%). The rest is vested.